src/tsd/GraphHandler.java

```diff
@@ -112,7 +112,7 @@ public void execute(final TSDB tsdb, final HttpQuery query) {
       if (uri.length() < 4) {  // Shouldn't happen...
         uri = "/";             // But just in case, redirect.
       } else {
-        uri = "/#" + uri.substring(3);  // Remove "/q?"
+        uri = "/#" + uri.substring(2);  // Remove "q?"
```

Comment:
Actually it's more complex than that.  I'm not sure it's possible to make this one relative, which might mean that TSD might need to know the path it's rooted under.  Maybe we should consider adding a setting so that TSD itself is serving all the URLs under a certain prefix.